.flaskenv.example

```diff
@@ -0,0 +1,3 @@
+FLASK_APP=app.py
+FLASK_ENV=development
+SECRET_KEY=your_secret_key_here
```

Comment:
For this project do we need a secret key?  No harm to leave a placeholder here tho.